bower.json

```diff
@@ -10,7 +10,8 @@
     "jquery": "https://github.com/jquery/jquery.git#2.1.4",
     "keyboardjs": "https://github.com/RobertWHurst/KeyboardJS.git#2.3.3",
     "knockout": "https://github.com/knockout/knockout.git#3.4.1",
-    "marked": "https://github.com/wireapp/marked.git#3fb540777bd042e7284d0a960729169d9efee508",
+    "linkifyjs": "https://github.com/nfrasser/linkify-shim.git#2.1.4",
+    "marked": "https://github.com/mythsunwind/marked.git#linkify",
```

Comment:
Please submit your changes from your `linkify` branch (of marked) to https://github.com/wireapp/marked